tests/framework/cli/test_cli.py

```diff
@@ -643,13 +629,13 @@ def test_run_with_params_in_config(
                 {"foo": "bar", "baz": "https://example.com"},
             ),
             ("foo:bar,baz:fizz buzz", {"foo": "bar", "baz": "fizz buzz"}),
-            ("foo:bar, foo : fizz buzz  ", {"foo": "fizz buzz"}),
             ("foo.nested:bar", {"foo": {"nested": "bar"}}),
-            ("foo.nested:123.45", {"foo": {"nested": 123.45}}),
+            ("foo.nested=123.45", {"foo": {"nested": 123.45}}),
             (
                 "foo.nested_1.double_nest:123.45,foo.nested_2:1a",
                 {"foo": {"nested_1": {"double_nest": 123.45}, "nested_2": "1a"}},
             ),
+            ("foo=bar", {"foo": "bar"}),
         ],
     )
     def test_run_extra_params(
```

Comment:
I think it would be clearer if we just have this test twice, once with `:` and once with `=` so we can clearly see both syntaxes work for all cases.

Reply:
The test with `:` is already there. I've moved the test parameters together to make it more readable.